exercises/templates/exercises/dl_digit_classifier_newmanager/exercise.html

```diff
@@ -0,0 +1,46 @@
+{% extends "react_frontend/exercise_base.html" %}
+{% load react_component %}
+
+{% block exercise_header %}
+{% endblock %}
+
+{% block react-content %}
+    {% react_component exercise/dl_digit_classifier_newmanager/FollowLineRR %}
```

Comment:
This line still points to FollowLineRR

Reply:
When I change the name of FollowLineRR to DigClassRR for example(the name of original file, its components as well as the name in exercise.html), webpage fails to load. Can you suggest what can be done?